test/deploy/kubernetes/manual/destroy.sh

```diff
@@ -1,77 +1,92 @@
+print_header "Deleting Cotrol Service - Cluster"
+kubectl delete -f $BASEPATH/external-services/headless_control_node.yaml --namespace $NAMESPACE
 
-# Delete Persistent Volume Claims
+# Delete Control Node (POD)
+print_header "Deleting Cotrol Node - Cluster"
+kubectl delete -f $BASEPATH/provisioner-pods/deployment_control_node.yaml --namespace $NAMESPACE
+
+# Delete Persistent Volume Claims for Block Devices (PVC)
 for NODE_INDEX in $(seq 1 $MAXNODES); do
     NODE_NAME="node"$NODE_INDEX;
-    print_header "Deleting Persistent Volume Claims - $NODE_NAME"
-    NODE_PVCS=$SCRIPT_DIR/volume-claims/blockdevices_claim_$NODE_NAME.yaml;
-    kubectl delete -f $NODE_PVCS --namespace $NAMESPACE;
+    print_header "Deleting Persistent Volume Claims for Block Devices - $NODE_NAME"
+    NODE_BVOL=$BASEPATH/volume-claims/block_volumes_$NODE_NAME.yaml;
+    kubectl delete -f $NODE_BVOL --namespace $NAMESPACE;
     sleep 2;
 done
 
-# Delete Persistent Volumes
+# Delete Persistent Volumes for Block Devices (PV)
 for NODE_INDEX in $(seq 1 $MAXNODES); do
     NODE_NAME="node"$NODE_INDEX;
-    print_header "Deleting Persistent Volumes - $NODE_NAME"
-    NODE_PVOL=$SCRIPT_DIR/persistent-volumes/block_devices_$NODE_NAME.yaml;
-    kubectl delete -f $NODE_PVOL --namespace $NAMESPACE;
+    print_header "Deleting Persistent Volumes for Block Devices - $NODE_NAME"
+    NODE_BDEV=$BASEPATH/persistent-volumes/block_devices_$NODE_NAME.yaml;
+    kubectl delete -f $NODE_BDEV --namespace $NAMESPACE;
     sleep 2;
 done
 
-# Delete Persistent Volume Claims for Block Volumes
+# Delete Persistent Volume Claims for Storage Devices (PVC)
 for NODE_INDEX in $(seq 1 $MAXNODES); do
     NODE_NAME="node"$NODE_INDEX;
-    print_header "Creating Persistent Volume Claims - $NODE_NAME"
-    NODE_BVOLC=$SCRIPT_DIR/volume-claims/blockvolumes_claim_$NODE_NAME.yaml;
-    kubectl delete -f $NODE_BVOLC --namespace $NAMESPACE;
+    print_header "Deleting Persistent Volume Claims for Storage Devices - $NODE_NAME"
+    NODE_SVOL=$BASEPATH/volume-claims/storage_volumes_$NODE_NAME.yaml;
+    kubectl delete -f $NODE_SVOL --namespace $NAMESPACE;
     sleep 2;
 done
 
-# Delete Persistent Volumes for Block Volumes
+# Delete Persistent Volumes for Storage Devices (PV)
 for NODE_INDEX in $(seq 1 $MAXNODES); do
     NODE_NAME="node"$NODE_INDEX;
-    print_header "Creating Persistent Volumes - $NODE_NAME"
-    NODE_BVOL=$SCRIPT_DIR/persistent-volumes/block_volumes_$NODE_NAME.yaml;
-    kubectl delete -f $NODE_BVOL --namespace $NAMESPACE;
+    print_header "Deleting Persistent Volumes for Storage Devices - $NODE_NAME"
+    NODE_SDEV=$BASEPATH/persistent-volumes/storage_devices_$NODE_NAME.yaml;
+    kubectl delete -f $NODE_SDEV --namespace $NAMESPACE;
     sleep 2;
-    kubectl get pv --namespace $NAMESPACE | grep $NODE_NAME;
 done
 
+# Remove remaining PVC
+kubectl delete pvc --all --force --namespace $NAMESPACE
+kubectl get pvc --namespace $NAMESPACE | grep $NODE_NAME
+
+# Remove remaining PV
```

Comment:
[SUGGESTION]:
It would be a good idea to add prints before each command so as to keep user informed about the operation being done.
e.g. here following print/echo can be added:
echo "Removing all PV for namespace:$NAMESPACE"